client/public/index.html

```diff
@@ -39,5 +38,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```

Comment:
Just so we keep this PR in scope, I'd say either reformat this file so that no changes are made, or simply disinclude it from this PR.